src/Go/boardAnalysis/scoring.ts

```diff
@@ -68,10 +69,12 @@ export function endGoGame(boardState: BoardState) {
       factionName &&
       statusToUpdate.winStreak % 2 === 0 &&
       Player.factions.includes(factionName) &&
-      statusToUpdate.favor < getMaxFavor()
+      statusToUpdate.rep < getMaxRep()
     ) {
-      Factions[factionName].setFavor(Factions[factionName].favor + 1);
-      statusToUpdate.favor++;
+      const currentFavor = Factions[factionName].favor;
+      const newFavor = calculateFavorAfterResetting(currentFavor, 1000);
+      Factions[factionName].setFavor(currentFavor + newFavor);
```

Comment:
Did you test this? CalculateFavorAfterResetting already does this addition.

Reply:
Good catch, fix'd